manager/container.go

```diff
@@ -342,10 +343,10 @@ func (c *containerData) housekeeping() {
 
 		// TODO(vmarmol): Export metrics.
 		// Run custom collectors.
-		nextCollectionTime, _, err := c.collectorManager.Collect()
-		if err != nil && c.allowErrorLogging() {
-			glog.Warningf("[%s] Collection failed: %v", c.info.Name, err)
-		}
+		// nextCollectionTime, _, err := c.collectorManager.Collect()
+		// if err != nil && c.allowErrorLogging() {
+		//	glog.Warningf("[%s] Collection failed: %v", c.info.Name, err)
+		//}
 
 		// Next housekeeping is the first of the stats or the custom collector's housekeeping.
```

Comment:
Given that we are now saving all stats in one place, it doesn't make sense to collect metrics at this level.
Either doing it in UpdateStats() or calling UpdateCustomStats() from housekeepingTick() should be fine.